src/App.js

```diff
@@ -31,7 +32,19 @@ export default function App() {
         <li>Abandon</li>
         <li>Punishment</li>
       </ol>
+      <div class="dropdown">
```

Comment:
Move this to a separate component, so that we can reuse and encapsulate the dropdown logic